spec/features/admin_can_edit_anything_spec.rb

```diff
@@ -53,4 +55,18 @@
     expect(page).not_to have_content("something comment something")
     expect(page).to have_content("comment deleted")
   end
+
+  scenario 'admin can make user a mod' do
+    sign_in_as(admin)
+    visit user_path(user)
+    click_link "make this user a mod"
+    expect(page).to have_content("revoke modship")
+  end
+
+    scenario 'admin can revoke modship' do
+    sign_in_as(admin)
+    visit user_path(mod)
+    click_link "revoke modship"
+    expect(page).to have_content("make this user a mod")
+  end
```

Comment:
`end` at 71, 2 is not aligned with `scenario 'admin can revoke modship' do` at 66, 4